Strategy11/formidable-forms
# Use FrmDb::get_results to avoid issue with repeated wp_unslash calls on a field

Fix https://github.com/Strategy11/formidable-pro/issues/5064

The issue seems to be caused by cached `FrmFields::get_all_for_form()` result being passed to `wp_unslash` more than once, causing backslashes to be removed totally. That is because `FrmFields::getAll` calls `FrmDb::getVar` which checks cache first (which is already passed to wp_unslash()), and then strips the slashes again before returning the result. In db backslashes are escaped when saved (`\\`) so that a single `wp_unslash` call should only strip a single slash at a time.

### Test steps

1. Add a text field with the format ^(21|22|23|24)\d{5}$.
2. Enable "Load and save form builder page with AJAX" in form settings.
3. Save the field and reload the form.
4. Select the field again.
5. Confirm that the format is not changed.
6. Preview the form and test the format validation inputting a text in the field.
